gen3-cli/tests/system/cmdb/alm/conftest.py

```diff
@@ -0,0 +1,48 @@
+from cot.backend.generate.cmdb import account
+
+
+conf = dict(
+    cmdb=dict(
+        tenant=dict(
+            tenant_id='tenant_id',
+            tenant_name='tenant',
+            domain_stem='domain.stem.com',
```

Comment:
I'd rather use some domain owned by us for demo purpose - Michael could advise

Reply:
codeontap.io would work, we own it and it’s our product

Reply:
Ok, no problem.